internal/parsers/db/put.go

```diff
@@ -36,12 +36,14 @@ func (db *CsvDB) Put(data []Data) error {
 		// Calculate a unique index for each key
 		keyString := key.CountryCode().String() + key.LocationCode().String()
 
-		if _, exists := uniqueKeys[keyString]; exists {
-			continue // Skip duplicates
+		if index, exists := uniqueKeys[keyString]; exists {
+			newRecordsLocode[index][5] = strconv.FormatFloat(float64(rec.Point.Latitude), 'f', -1, 32)
```

Comment:
it seems like the second (at least) usage of magic `5` and `6`  number, maybe add a cost for them?

Reply:
Added constants in pkg/locodedb/utils.go. Is it good idea to store them there?

Reply:
yes, but usually every exported thing must have a comment about what it is